<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>USTB选课系统V0.10</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/20150816065626967.ico</normaloff>../../../Downloads/20150816065626967.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>登陆</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>211</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>用户名:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="usrNameEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>211</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>密  码   :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pswdEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>USTB选课系统V0.10</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="savePrpCheckBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>286</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>软件相关</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_notice"/>
    <addaction name="action_update"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>检查更新</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>关于本软件</string>
   </property>
  </action>
  <action name="action12321">
   <property name="text">
    <string>12321</string>
   </property>
  </action>
  <action name="action_update">
   <property name="text">
    <string>检查更新</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_notice">
   <property name="text">
    <string>注意事项</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
